### ISSUE TITLE
app/tmp/main.go: Error: cannot refer to unexported name — if a private structure defined with "*revel.Controller" inside

### ISSUE THREAD

**Issue by xaionaro** [revel/revel#1241]
If I create file `app/helpers/controllers/controller.go` and add there:
```
package controllersHelpers

type controller struct {
        *revel.Controller
        EventSQL *models.EventScope
}
```

Then I get error:
```
gopath/src/?/app/tmp/main.go:68:29: cannot refer to unexported name controllersHelpers.controller
```

This type shouldn't be used for any code-generation by revel. It's private :)

I need this helper to bypass this issue: https://github.com/revel/revel/issues/1239 :)

**Comment by lujiacn** [comment]
try
```go
package controllersHelpers

type controller struct {
        *revel.Controller
        *models.EventScope
}
```

**Comment by xaionaro** [comment]
@lujiacn, doesn't help. The same error.

**Comment by notzippy** [comment]
Quite right, this is a bug, unexported types should be ignored

**Comment by xaionaro** [comment]
> I need this helper to bypass this issue: #1239 :)

It's not required anymore.

**Comment by notzippy** [comment]
But I got a fix revel/cmd#100
